runtime/lua/vim/provider/ruby.lua

```diff
@@ -0,0 +1,61 @@
+local M = {}
+local s_err ---@type string?
+local s_host ---@type string?
```

Comment:
technically this is `integer`. (I also missed this too for `perl` and `python` providers)